src/index.ts

```diff
@@ -1,9 +1,9 @@
-import { name as pkgName } from '../package.json';
 import { NodePath, PluginObj, PluginPass } from '@babel/core';
 import debugFactory from 'debug';
 import * as util from '@babel/types';
 import template from '@babel/template';
 
+const pkgName = 'babel-plugin-explicit-exports-references';
```

Comment:
I had to change this to get the `tsconfig.build.json` to work (since the root is now `src/`).